workflows/epigenetics/hic-hicup-cooler/chic-fastq-to-cool-hicup-cooler.ga

```diff
@@ -1,6 +1,6 @@
 {
     "a_galaxy_workflow": "true",
-    "annotation": "This workflow take as input a collection of paired fastq. It uses HiCUP to go from fastq to validPair file. The pairs are filtered for MAPQ and for the region captured. Then, they are sorted by cooler to generate a tabix dataset. Cooler is used to generate a balanced cool file to the desired resolution.",
+    "annotation": "This workflow processes paired fastq files with HiCUP to create validPair files. It filters pairs by MAPQ and captured region, then sorts them with cooler to generate tabix datasets and balanced cool files at desired resolution.",
```

Comment:
It looks like some of the annotations were not updated, but this update look good to me. Considering that's what we pull into BRC maybe we should start with updating the name and annotation fields before we tweak the readme ?